packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -148,6 +149,22 @@ describe('ActorQueryOperationFilterSparqlee', () => {
       await new Promise<void>(resolve => output.bindingsStream.on('error', () => resolve()));
     });
 
+    it('Should log warning for an expressionError', async() => {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      const logWarnSpy = jest.spyOn(<any> actor, 'logWarn');
+      // eslint-disable-next-line no-import-assign
+      Object.defineProperty(sparqlee, 'isExpressionError', { writable: true });
+      (<any> sparqlee).isExpressionError = jest.fn(() => true);
+      const op = { operation: { type: 'filter', input: {}, expression: erroringExpression }};
+      const output: IActorQueryOperationOutputBindings = <any> await actor.run(op);
+      await new Promise<void>(resolve => output.bindingsStream.on('end', () => resolve()));
+      expect(logWarnSpy).toHaveBeenCalled();
```

Comment:
If you use `toHaveBeenCalledTimes(1)`, you can remove the following loop to just check `logWarnSpy.mock.calls[0][2]`.

Reply:
This doesn't work because the error gets called 3 times?

Reply:
Ah yes, because `bindingsStream` contains 3 bindings. Nvm then :-)

But do check that `toHaveBeenCalledTimes(3)` then.